X-ray Absorption Fine Structure/plans.py

```diff
@@ -60,7 +63,10 @@ def rocking_curve(start=-0.10, stop=0.10, nsteps=101, choice="peak"):
     """
 
     title = "I0 signal vs. DCM 2nd crystal pitch"
-    @subs_decorator(LivePlot("I0", pitch.name))
+
+    fig, ax = plt.subplots(num=title)
```

Comment:
Is this clever or a mistake?

Reply:
clever.  This is how you give it the name and title you want.